examples/axp_optimizer_example.py

```diff
@@ -0,0 +1,143 @@
+import time
+import os
+import uuid
+import pandas as pd
+
+from fds.analyticsapi.engines import ApiException
+from fds.analyticsapi.engines.api.axp_optimizer_api import AXPOptimizerApi
+from fds.analyticsapi.engines.api_client import ApiClient
+from fds.analyticsapi.engines.configuration import Configuration
+from fds.analyticsapi.engines.model.axioma_equity_optimization_parameters_root import AxiomaEquityOptimizationParametersRoot
+from fds.analyticsapi.engines.model.axioma_equity_optimizer_strategy import AxiomaEquityOptimizerStrategy
+from fds.analyticsapi.engines.model.optimizer_account import OptimizerAccount
+from fds.analyticsapi.engines.model.axioma_equity_optimization_parameters import AxiomaEquityOptimizationParameters
+from fds.analyticsapi.engines.model.optimizer_output_types import OptimizerOutputTypes
+from fds.analyticsapi.engines.model.optimizer_trades_list import OptimizerTradesList
+from fds.analyticsapi.engines.model.optimization import Optimization
+from fds.protobuf.stach.extensions.StachExtensionFactory import StachExtensionFactory
+from fds.protobuf.stach.extensions.StachVersion import StachVersion
+
+from urllib3 import Retry
+
+host = "https://api.factset.com"
+username = os.environ["ANALYTICS_API_USERNAME_SERIAL"]
```

Comment:
Can you please change the environment variable to `ANALYTICS_API_QA_USERNAME_SERIAL`?

Reply:
A person reading this code has the perspective of them being a user, their concept of QA would differ from our concept of QA. I worry that breaks abstraction.